### PR TITLE
SIG-storage charter

### PR DESCRIPTION
**Pull request by saad-ali**
Charter for @kubernetes/sig-storage-misc 

/assign @childsb for initial pass. Will assign to TOC for approval once we have internal SIG storage consensus.

Also closes https://github.com/kubernetes/community/issues/2467

### CODE REVIEW

File: sig-storage/charter.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+### In scope
+
+Some notable examples of features owned by SIG Storage:
+
+* Persistent Volume Claims and Persistent Volumes
+* Storage Classes and Dynamic Provisioning
+* Kubernetes volume plugins
+* Container Storage Interface (CSI)
+* Secret, ConfigMap, DownwardAPI Volumes
```
Comment (j-griffith) [inline sig-storage/charter.md]: Don't know if this should be more explicit that its Secrets, ConfigMap etc related to Volumes?  I know that's pretty implicit due to the doc and such but just a nit.

Comment (msau42) [reply sig-storage/charter.md]: Do we want to mention enptydir volumes too?

Comment (saad-ali) [reply sig-storage/charter.md]: Done!

File: sig-storage/charter.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### In scope
+
+Some notable examples of features owned by SIG Storage:
+
+* Persistent Volume Claims and Persistent Volumes
+* Storage Classes and Dynamic Provisioning
+* Kubernetes volume plugins
+* Container Storage Interface (CSI)
+* Secret Volumes, ConfigMap Volumes, DownwardAPI Volumes, EmptyDirVolumes (co-owned with SIG-Node)
```
Comment (msau42) [inline sig-storage/charter.md]: nit: space between EmptyDir and Volumes

**Comment by spiffxp** [comment]
Can you split the sigs.yaml change out of this? @brendandburns [is listed as the primary reviewer for this](https://docs.google.com/spreadsheets/d/1i_8oM1KlmXo90zIIgNTdOg-M6xesTggFLnTkgmscMlI/edit#gid=0) and is out next week.  I'd like to get the sigs.yaml piece merged sooner.

**Comment by saad-ali** [comment]
@spiffxp Done: https://github.com/kubernetes/community/pull/2519

**Comment by spiffxp** [comment]
/cc @michelleN 
expressed an interest in reviewing as well

File: sig-storage/charter.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+### Deviations from [sig-governance]
+
+SIG Storage does not have separate tech leads: SIG Storage chairs serve as tech leads.
+
+### Subproject Creation
+
+SIG Storage delegates subproject approval to Technical Leads. See [Subproject creation - Option 1].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[sigs.yaml]: https://github.com/kubernetes/community/blob/master/sigs.yaml#L1454
```
[the remark this answers is not in the document]
Comment (spzala) [reply sig-storage/charter.md]: @saad-ali thanks, not sure if I am missing something here and if so please ignore my comment but what I noticed is, 
1. The charter.md seems not using sigs.yaml anywhere, so I don't think related link is needed
2. Even if we are using it, then referencing it by line number is not good as numbers will change, the charter template is already modified,  https://github.com/kubernetes/community/commit/e0d60fdd7cfdafb71335c9501c458278bf7fdf97

**Comment by spzala** [comment]
@saad-ali, :-) np, looks good to me now!

**Pull request merged**